routes/dms.js

```diff
@@ -10,12 +10,22 @@ const https = require("https")
 const http = require("http")
 const { callbackPromise } = require('nodemailer/lib/shared')
 const logger = require('../utils/logger')
+const { param } = require('express-validator')
+const createDOMPurify = require('dompurify');
+const { JSDOM } = require('jsdom');
+
 module.exports = function () {
   const router = express.Router()
   const Model = new dms.DmsModel(config)
 
   // -----------------------------------------------
   // Redirects
+ 
+  const xssSenitize = (value) => {
```

Comment:
@sagargg "senitize" is not correct spelling

Reply:
@anuveyatsu  spelling corrected.